autoload/leaderf/python/leaderf/commandExpl.py

```diff
@@ -73,16 +75,20 @@ def _defineMaps(self):
     def _acceptSelection(self, *args, **kwargs):
         """"""
         cmd = args[0]
-        try:
-            lfCmd(cmd)
-            lfCmd("call histadd(':', '%s')" % escQuote(cmd))
-        except vim.error as e:
-            error = lfEncode(str(repr(e)))
-            if "E471" in error:
-                # Arguments mandatory
-                lfCmd("call feedkeys(':%s ')" % escQuote(cmd))
-            else:
-                lfPrintError(e)
+
+        if self._getExplorer()._run_immediately:
```

Comment:
Can be replaced by 
`if "--run-immediately" in self._arguments:`
So self._run_immediately can be removed from CommandExplorer.

Reply:
I didn't know there were a `self._arguments`.
Thanks!